.github/workflows/mypy.yaml

```diff
@@ -35,15 +35,15 @@ jobs:
           cache-environment-key: environment-${{ steps.date.outputs.date }}
           cache-downloads-key: downloads-${{ steps.date.outputs.date }}
           create-args: >-
-            python=3
+            python=3.12
             rdkit=2023.09.5
```

Comment:
we should be able to unpin rdkit with a little work: https://github.com/OpenFreeEnergy/openfe/issues/1439